PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/layers/transmitter.cpp

```diff
@@ -0,0 +1,9 @@
+#include "transmitter.h"
+/**
+ * @brief Sends an InitFC2 message to the receiver.
+ * @param void
+ * @return void
+*/
+void Transmitter::sendInitFC2(void) {
+	this->layers.sendNOpFlit(this->globals.P_CREDITS, this->globals.NP_CREDITS, this->globals.CPP_CREDITS, Dllp::DllpType::initFC2);
```

Comment:
I think the function would be spelled either NOP of Nop. So let's wait until it's implemented